zqd/handlers.go

```diff
@@ -274,7 +311,17 @@ func handlePcapPost(c *Core, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	op, err := ingest.NewPcapOp(ctx, s, req.Path, c.ZeekLauncher)
+	pspace, ok := s.(ingest.PcapSpace)
+	if !ok {
+		respondError(c, w, r, zqe.E(zqe.Invalid, "space does not support pcap import"))
```

Comment:
Can we change line 255 to have the same error message?

Reply:
The line 255 I see is for the warning about zeek not being present; I'd rather keep that in place since it describes a different scenario (zqd not having a configured zeek vs an individual space not capable of handling pcaps).

Reply:
ok